telemetry/definitions/commonDefinitions.json

```diff
@@ -3538,8 +3538,10 @@
                     "type": "codeTransformSessionId",
                     "required": true
                 },
-                { "type": "result", "required": true },
-                { "type": "reason", "required": true }
```

Comment:
We do emit result and reason for this metric, and I have no plan to change this, but I removed these here because I noticed that they are missing in many other metrics, so I want to be consistent.

Reply:
EDIT: Added them back.

Reply:
Moving forward we will want to have reason/result in all of our metrics, but since we are already working on revamping the metric schema and the majority of these metrics will be replaced, I think it's fine to not back-fill the fields at the moment.

Reply:
Not just relate to this case
if any field is required true, that means it needs to be populated. if you are concerned about consistency, required:true is nice way to force it.